client/src/components/PostCard.tsx

```diff
@@ -0,0 +1,111 @@
+import { Card, Space } from 'antd';
+import React from 'react';
+import { Carousel } from 'antd';
+import type { CollapseProps } from 'antd';
+import { Collapse } from 'antd';
+import { Share, Heart, MessageText1, Whatsapp, Call, Send } from 'iconsax-react';
+import { Col, Row } from 'antd';
+import { Popover } from 'antd';
+import '../styles/post-card.css'
+import { Link } from 'react-router-dom';
+
+
+const petDetails = (<>
+    <Row>
+        <Col span={12} className='collapse-details'><p>name: cat</p></Col>
+        <Col span={12} className='collapse-details'><p>age: 5</p></Col>
+    </Row>
+    <Row>
+        <Col span={12} className='collapse-details'><p>gender: cat</p></Col>
+        <Col span={12} className='collapse-details'><p>petType: cat</p></Col>
+    </Row>
+    <Row>
+        <Col span={12} className='collapse-details'><p>healthStatus: cat</p></Col>
+        <Col span={12} className='collapse-details'><p>adoptionStatus: cat</p></Col>
+    </Row>
+</>)
+
+const items: CollapseProps['items'] = [
+    {
+        key: '1',
+        label: 'Pet Details',
+        className: 'collapse-details',
+        children: <>{petDetails}</>,
+    }
+];
+const content = (
+    <div className='content-pop'>
+        <a href='https://wa.me/<number>' className='contact-pop' ><Whatsapp /> +970599999999</a>
+        <a href='https://t.me/<number>' className='contact-pop' ><Call /> +970599999999</a>
+    </div>
+);
+
+const PostCard: React.FC = () => {
+    return <Space direction="vertical" size={16}>
+        <Card className='card'>
+            <div className='post-header'>
+                <div className='user-post-container'>
+                    <img src="https://cdn.discordapp.com/attachments/1113720733860888597/1121405281147027526/IMG_20201207_144829.jpg" alt="user-img" className='user-img' />
+                    <div className='user-post-info'>
+                        <Link to={'#'} className='username'>Haitham</Link>
+                        <h4 className='date'>13 min ago</h4>
+                    </div>
+                </div>
+                <div className='label'>
+                    <div className='top-label' ></div>
+                    <p className='label-content'>Adoption</p>
+                    <div className='down-label'></div>
+                </div>
+            </div>
+            <Carousel className='carousel'>
+                <div>
+                    <img src="https://cdn.discordapp.com/attachments/1113720733860888597/1121405281147027526/IMG_20201207_144829.jpg" alt="image" className='img' />
```

Comment:
Create a new component in components file called `Image` which you will use antd `Image` component with `preview false